src/lib/integration.spec.ts

```diff
@@ -1,4 +1,4 @@
-// eslint-disable-next-line import/no-extraneous-dependencies
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Instead of adding these comments inside of spec files, create an overwrite rule in the eslint config e.g.:

```
module.exports = {
...
  overrides: [
    {
      files: ['*.{spec,test}.{ts,tsx,js,jsx}', 'setupTests.{ts,js}', '**/testHelpers/**/*'],
      rules: {
        'import/no-extraneous-dependencies': 0,
      },
    },
  ]
...